engine/crates/parser-postgresql/src/registry/types.rs

```diff
@@ -66,8 +66,15 @@ fn generate_types(
                 key: column.database_name().to_string(),
             });
 
-            if column.database_type().is_enum() {
-                field.resolver = field.resolver.and_then(Transformer::RemoteEnum);
+            let extra_transformer = match column.database_type() {
+                DatabaseType::Scalar(ScalarType::Bytea) => Some(Transformer::BytesToBase64),
+                DatabaseType::Scalar(ScalarType::ByteaArray) => Some(Transformer::ByteArrayToBase64Array),
```

Comment:
Is the extra a on the end of `Bytea` & `ByteaArray` a typo?

Reply:
No, the type is `bytea`.